subsys/bluetooth/shell/bt.c

```diff
@@ -1542,6 +1542,8 @@ static bool adv_param_parse(size_t argc, char *argv[],
 			param->options |= BT_LE_ADV_OPT_FORCE_NAME_IN_AD;
 		} else if (!strcmp(arg, "low")) {
 			param->options |= BT_LE_ADV_OPT_DIR_MODE_LOW_DUTY;
+		} else if (!strcmp(arg, "dir-rpa")) {
+			param->options |= BT_LE_ADV_OPT_DIR_ADDR_RPA;
```

Comment:
seems this should be in a different commit. or just update the commit message, but it's a functional change not just help text.